### PR TITLE
Feature/search api

### PR DESCRIPTION
**Pull request by mstrutt**

### CODE REVIEW

File: app/assets/js/controllers/search-results.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+SearchResultsController.resolve = /* @ngInject */ {
+	searchResponse: function($stateParams, searchService, baseSearchParams) {
+		var searchParams = angular.copy(baseSearchParams);
+		searchParams.query = $stateParams.query;
+
+		return searchService
+			.getResults(searchParams)
+			.then(function(response) {
```
Comment (toboid) [inline app/assets/js/controllers/search-results.js]: Don't think you need the `then` as it's returning the same value?

Comment (mstrutt) [inline app/assets/js/controllers/search-results.js]: no, it's not needed. I'd just added it so I could add a breakpoint to the promise resolving. Nice spot :)

**Comment by toboid** [comment]
could you rebase please

**Comment by mstrutt** [comment]
rebased :)

**Pull request merged**